wee_slack.py

```diff
@@ -4000,22 +4001,38 @@ def command_thread(data, current_buffer, args):
 
 command_thread.completion = '%(threads)'
 
+
 @slack_buffer_required
 @utf8_decode
 def command_reply(data, current_buffer, args):
     """
-    /reply <count/message_id> <text>
-    Reply in a thread on the message. Specify either the message id
-    or a count upwards to the message from the last message.
+    /reply [-alsochannel] <count/message_id> <text>
+    Reply in a thread on the message. Specify either the message id, a count
+    upwards to the message from the last message, or, if you are already in a
+    thread, nothing.
     """
     channel = EVENTROUTER.weechat_controller.buffers[current_buffer]
+    parts = args.split(None, 1)
+    if parts[0] == "-alsochannel":
+        args = parts[1]
+        broadcast = True
+    else:
+        broadcast = False
+
     try:
-        msg_id, text = args.split(None, 1)
+        if isinstance(channel, SlackThreadChannel):
+            text = args
+        else:
+            msg_id, text = args.split(None, 1)
     except ValueError:
-        w.prnt('', 'Usage: /reply <count/id> <message>')
+        w.prnt('', 'Usage: /reply [-alsochannel] <count/id> <message>')
         return w.WEECHAT_RC_OK_EAT
 
-    msg = get_msg_from_id(channel, msg_id)
+    if isinstance(channel, SlackThreadChannel):
+        msg = channel.parent_message
+    else:
+        msg = get_msg_from_id(channel, msg_id)
```

Comment:
I think this is a bit more readable if you combine the two checks for thread channel, like this:

```
    if isinstance(channel, SlackThreadChannel):
        text = args
        msg = channel.parent_message
    else:
        try:
            msg_id, text = args.split(None, 1)
        except ValueError:
            w.prnt('', 'Usage: /reply [-alsochannel] <count/id> <message>')
            return w.WEECHAT_RC_OK_EAT
        msg = get_msg_from_id(channel, msg_id)
```